release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -341,6 +341,18 @@ The Cloud Credential Operator utility (`ccoctl`) now creates secrets that use re
 
 With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
 
+[discrete]
+[id="ocp-4-12-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
+
+See xref:../authentication/understanding-and-managing-pod-security-admission.adoc#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission[Identifying pod security violations] for information about how to find which workloads are causing pod security violations.
+
+// TODO: What "call to action" can we say after this for people to prep for 4.13 enforcement? I don't see any docs on what to actually do when you have a workloads in violation
```

Comment:
@stlaz Thoughts on what we should tell people to act on now for these violations that they experience?

We're adding a section on how to find what is causing the violations (via #50631), but don't really have anything on how people are supposed to resolve violations. So I'm not sure what to say here for that. Thoughts?